### ISSUE TITLE
Shortcuts

### ISSUE THREAD

**Issue by LiviLou** [Bill13579/tabby#47]
Maybe it's just me... but the keyboard shortcut to open Tabby selecting Tab (Ctrl+Shift+X) does nothing. I can open Tabby selecting Search but then the arrows and such does nothing until I move the cursor up hovering a tab first.

**Comment by Bill13579** [comment]
Yup, I can confirm that this is happening on Windows. Will be fixed in the next release

**Comment by keeganwitt** [comment]
It might also be good to document the shortcuts in the settings somewhere.

**Comment by Marvelous-Deducer** [comment]
wow i finally found this thread!
me too .. ctrl shift x does nothing
tabby is amazing!
with just shortcuts you can navigate the entire set of tabs with just keyboard!!
It's more useful than mouse because you can't see the entire title on a tab in firefox

can't wait for the next release!

**Comment by cptcontainers** [comment]
Can confirm also on linux (fedora) + firefox the ctrl shift x does not work so I remapped it to ctrl shift u for now. But potentially worth remapping the default if many others also have issues with it

**Comment by Bill13579** [comment]
> Can confirm also on linux (fedora) + firefox the ctrl shift x does not work so I remapped it to ctrl shift u for now. But potentially worth remapping the default if many others also have issues with it

Yeah I'll remap it to Alt+Period by default in the next release, hopefully that will fix the issue for most
